src/transforms/regex_parser.rs

```diff
@@ -35,7 +35,7 @@ pub struct RegexParserConfig {
 }
 
 inventory::submit! {
-    TransformDescription::new::<RegexParserConfig>("regex_parser")
+    TransformDescription::new::<RegexParserConfig>("regex")
```

Comment:
Is this correct? Everywhere else I still see it named `regex_parser`?

Reply:
Ah I see `transform_type` returns `regex` as well. Should we make this consistent and rename it everywhere? For example, Serde still expects the `regex_parser` tag, it seems.

Reply:
I see the documentation uses `regex_parser`, so let's change `transform_type` to return `regex_parser`.